pkg/envtest/server.go

```diff
@@ -322,6 +333,20 @@ func (te *Environment) startControlPlane() error {
 	return nil
 }
 
+func (te *Environment) waitForDefaultNamespace(config *rest.Config) error {
+	cs, err := client.New(config, client.Options{})
+	if err != nil {
+		return fmt.Errorf("unable to create client: %w", err)
+	}
+	// It shouldn't take longer than 5s for the default namespace to be brought up in etcd
+	return wait.PollUntilContextTimeout(context.TODO(), time.Millisecond*500, time.Second*5, true, func(ctx context.Context) (bool, error) {
```

Comment:
```suggestion
	return wait.PollUntilContextTimeout(context.TODO(), time.Millisecond*50, time.Second*5, true, func(ctx context.Context) (bool, error) {
```

Maybe let's do this a bit quicker so we don't lose a full 500ms if the namespace doesn't show up immediately

Reply:
Yep, makes sense. Updated.